catalog/ui/search-ui/standard/src/main/webapp/js/view/MetacardDetail.view.js

```diff
@@ -102,9 +106,35 @@ define([
                     $('#nextRecord', this.$el).addClass('disabled');
                 }
             },
+            getNearby: function () {
+                if (!nearbyChecked) {
+                    $('#nearby').html("");
```

Comment:
So we don't normally do this sort of thing when we're working with backbone. My suggestion would be to create a view that can represent this "nearby" section of the metacard and a model that has the distance, direction, name, etc. and pass that to the view. Then the view would just display the model and pop the text in there for you without needing to use jquery at all. To take it even further, you could get rid of the ajax call below and push all of that to the model so that you could create the model with a default value (like the WKT) and call sync on the model to have it go off and load itself up with all those fields. Then in this view, all you'd need to actually do is just display the NearbyView in a new region and the data would just magically show up.